.github/workflows/ci.yaml

```diff
@@ -0,0 +1,117 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'main'
+
+env:
+    SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+jobs:
+    composer-validate:
+        name: "Composer validate (${{ matrix.php-version }})"
+
+        runs-on: ubuntu-16.04
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '7.4'
+
+        steps:
+            -
+                name: "Checkout code"
+                uses: "actions/checkout@v2"
+
+            -
+                name: "Install PHP"
+                uses: "shivammathur/setup-php@v2"
+                with:
+                    coverage: "none"
+                    php-version: "${{ matrix.php-version }}"
+                    tools: "composer:v2"
+
+            -
+                name: "Validate composer.json"
+                run: "composer validate --strict --no-check-lock"
+
+            -
+                name: "Composer install"
+                uses: "ramsey/composer-install@v1"
+                with:
+                    composer-options: "--no-scripts"
+
+    tests:
+        needs:
+            - "composer-validate"
+
+        name: "PHP ${{ matrix.php-version }} + ${{ matrix.dependencies }}"
+
+        runs-on: ubuntu-16.04
+
+        continue-on-error: ${{ matrix.allow-failures }}
```

Comment:
why making this a matrix property when you always use `false` for it (which is even the default behavior on github). this is only making the config more confusing IMO.

thus, `continue-on-error` is *not* the equivalent of Travis's `allow-failures`, as it also continues running further steps (so if installing PHP breaks, it will still try to run composer after that for instance)

Reply:
I am using this config all over the place and it works for me like I expect.
I do not agree here, if you check https://github.com/OskarStark/flex/runs/1426102465?check_suite_focus=true for example, composer install step is breaking and all following steps are skipped, which is what I want to achieve:
![CleanShot 2020-11-27 at 13 05 59](https://user-images.githubusercontent.com/995707/100447705-5a769e80-30b1-11eb-9eac-0ce180a03558.png)

Reply:
@OskarStark you are showing a case with `continue-on-error: false` (which is the default behavior on Github and the only one used in this config).

What I'm saying is that setting `allow-failures: true` in your config (not done for now, which is why the matrix is useless for it) will *not* do the equivalent of the Travis one (and will *not* do what your screenshot shows)

Reply:
You are right I only use `false`, I removed it 👍